app/src/components/account/EmailAuthentication.vue

```
@@ -40,6 +44,9 @@ export default {
     goToNotifications() {
       this.$router.push({ name: "notifications" }).catch((err) => {})
     },
+    goToSignInModal() {
+      this.$router.push({ name: "sign-in-modal" }).catch((err) => {})
```

This is not needed and in this form would hide errors...

---

what do you mean by "hide errors"?